src/Storages/MergeTree/MergeTreeSettings.h

```diff
@@ -65,6 +66,7 @@ struct Settings;
     M(UInt64, min_age_to_force_merge_seconds, 0, "If all parts in a certain range are older than this value, range will be always eligible for merging. Set to 0 to disable.", 0) \
     M(UInt64, merge_tree_enable_clear_old_broken_detached, false, "Enable clearing old broken detached parts operation in background.", 0) \
     M(Bool, remove_rolled_back_parts_immediately, 1, "Setting for an incomplete experimental feature.", 0) \
+    M(CleanDeletedRows, clean_deleted_rows, CleanDeletedRows::Never, "Is the Replicated Merge cleanup has to be done automatically at each merge or manually (possible values are \"Always\"/\"Never\"(default))", 0) \
```

Comment:
Would "Automatically" be a more descriptive name for "Always"?

Reply:
In my opinion, "Automatically" is too generic. With `Always` I truly want to oppose to `Never`. 
As discussed in this PR, more modes could be added in the future an 'automatically' could mean a lot of behaviour. Here, we are applying the clean up at every merge.